### PR TITLE
fix(pages): rename pages, fix dom nesting errors

### LINKED ISSUE #127: Change page header from "Astro feed" to "Global Feed"
**STR:**
- Open the Global Feed page

**Expected result:**
- Page header is "Global Feed"

**Actual result:**
- Page header is "Astro feed"

---

image

### PR DESCRIPTION
**Pull request by VERZUOL1**

### CODE REVIEW

File: astro_2.0/features/Feed/Feed.module.scss [hunk 1]
```diff
@@ -16,7 +16,7 @@
   @include wide-background;
   padding: 56px 0 0;
   position: sticky;
-  top: 64px;
+  top: 72px;
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/Feed/Feed.module.scss:19]: We should adjust it with proposal type filter.

Comment (AlexeyPonomarenko) [inline astro_2.0/features/Feed/Feed.module.scss:19]: We should adjust it with proposal type filter.

**Review by AlexeyPonomarenko — CHANGES_REQUESTED** [review]

File: astro_2.0/features/Feed/Feed.module.scss (hunk 1 above)
Comment (VERZUOL1) [reply astro_2.0/features/Feed/Feed.module.scss:19]: done

File: astro_2.0/features/Proposals/components/ProposalCategoryFilter/ProposalCategoryFilter.module.scss [hunk 2]
```diff
@@ -21,7 +21,7 @@
     margin: 0;
     margin-bottom: 32px;
     max-width: initial;
-    top: 168px;
+    top: 176px;
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/Proposals/components/ProposalCategoryFilter/ProposalCategoryFilter.module.scss:24]: There is still an issue on the search results page in the "Proposal" tab, I'm afraid.

**Review by AlexeyPonomarenko — CHANGES_REQUESTED** [review]

File: astro_2.0/features/Proposals/components/ProposalCategoryFilter/ProposalCategoryFilter.module.scss (hunk 2 above)
Comment (VERZUOL1) [reply astro_2.0/features/Proposals/components/ProposalCategoryFilter/ProposalCategoryFilter.module.scss:24]: fixed

**Review by AlexeyPonomarenko — APPROVED** [review]

**Pull request merged**